MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -655,7 +577,7 @@ private StreamInfo BuildVideoItem(MediaSourceInfo item, VideoOptions options)
                 throw new ArgumentNullException(nameof(item));
             }
 
-            StreamInfo playlistItem = new StreamInfo
+            StreamInfo outputStream = new StreamInfo
```

Comment:
```suggestion
            StreamInfo outputStreamInfo = new StreamInfo
```

This object isn't a `Stream`

Reply:
Addressed same naming convention in other functions as well :+1: